MobileCenterCrashes/MobileCenterCrashes/Internals/Model/MSErrorAttachment.h

```diff
@@ -1,5 +1,7 @@
 #import <Foundation/Foundation.h>
 
+NS_ASSUME_NONNULL_BEGIN
```

Comment:
When I open Xcode I see, that MSErrorAttachment.h is in folder "MobileCenterCrashes", but physically it is in folder "Model". And MSErrorAtachment.h is a public header.